code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -475,6 +482,9 @@
 				if(2)
 					L.receive_damage(0,5)
 					Stun(100)
+			if((EASYDISMEMBER in L.owner.dna.species.species_traits) && L.body_zone != "chest")
+				if(prob(20))
+					L.dismember(BRUTE)

```

Comment:
Are these limbs still taking damage?

Also, if its a human with a mechanical heart.... does it try to dismember your torso?

Reply:
Yes. They take damage before this is checked

A human with a mechanical heart has no relevance: They do not have easy dismember. Additionally that's ORGAN_ROBOTIC, not BODYPART_ROBOTIC (all this applies to)

Additionally this ignores the torso, because getting EMP'd and spilling your posibrain for instant death is silly.